cmd/crc/cmd/start.go

```diff
@@ -35,21 +35,21 @@ func init() {
 	addOutputFormatFlag(startCmd)
 
 	flagSet := pflag.NewFlagSet("start", pflag.ExitOnError)
-	flagSet.StringP(crcConfig.Bundle, "b", constants.GetDefaultBundlePath(crcConfig.GetPreset(config)), "The system bundle used for deployment of the OpenShift cluster")
+	flagSet.StringP(crcConfig.Bundle, "b", constants.GetDefaultBundlePath(crcConfig.GetPreset(config)), "The system bundle used for provision virtual machine")
```

Comment:
`to provision the virtual machine`.

Reply:
We decided that the word "instance" works best

/cc: @robin-owen can chime in, as that is what we will do for the docs.